### PR TITLE
Grinch ability, ability fixes, and gimmick antags for admins

### PR DESCRIPTION
**Pull request by NightmareChamillian**
## About the PR 
Adds ability sprites for the grinches, fixes a parenting issue with the flamethrower abilities not showing an icon, adds a new sting ability for one of the new humans, and two gimmick carbons, for testing purposes and admins to terrify the crew with:

1) Cat Mascot, a hit and run character with a fire axe and wresting abilities. Meant to kick the shit out of random people, then vanish into a new department, rinse and repeat.
2) Angel mascot, a weird looking person with crowd control abilities, a shotgun and a more potent knockout sting. 

## Why's this needed? 
The gimmick humans were primarily added as prefab carbons with some form or another of AA for the sake of testing things without manually spawning all the various equipment in, but were morphed into antagonizing roles for the sake of entertainment. Ability fixes and respites came after I realized there was work that needed to be done in the respective fields, but are the actual motive for the PR itself. However, I still very much do want to see someone get kicked down an entire hallway by a man in a cat suit on multiplayer.

### CODE REVIEW

**Comment by mc-oofert** [comment]
changelog is still default

File: code/mob/living/carbon/human/gimmick.dm [hunk 1]
```diff
@@ -119,6 +119,76 @@
...
 		bioHolder.mobAppearance.underwear = "boxers"
 		. = ..()
+/mob/living/carbon/human/angel
+//I have no clue if this is going to work or not
+	New()
+		..()
+		SPAWN_DBG(0)
+			src.real_name = "??????"
+			src.add_ability_holder(/datum/abilityHolder/gimmick)
+			src.nodamage = 0
```
Comment (Zonespace27) [inline code/mob/living/carbon/human/gimmick.dm:129]:
```suggestion
			src.nodamage = FALSE
```
Isn't `nodamage` false by default on humans, anyhow?

File: code/obj/item/card.dm [hunk 2]
```diff
@@ -165,6 +165,16 @@ GAUNTLET CARDS
 			touched = TRUE
 			logTheThing("station", user, null, "is the first non-Captain to pick up [src] at [log_loc(src)]")
 
+/obj/item/card/id/catman
+	registered="Donk Co Super Duper Happy Funtime Friendly Cat Mascot"
+	assignment="Mascot"
```
Comment (Zonespace27) [inline code/obj/item/card.dm:170]:
```suggestion
	assignment = "Mascot"
```

File: code/mob/living/carbon/human/gimmick.dm (hunk 1 above)
Comment (NightmareChamillian) [reply code/mob/living/carbon/human/gimmick.dm:129]: Yes! That's remnant code from the jester that I changed, and (theoretically) should/can be deleted without any problems. Furthermore, the game doesn't seem to care whether or not it's set to 0 or false, and both exhibit the same behavior.

**Comment by Tarmunora** [comment]
atomize this

**Review by ZeWaka — CHANGES_REQUESTED** [review]
^

**Pull request closed without merging**